### PR TITLE
[PERFSCALE-3494] Adds AWS OVN IPSEC for perfscale

### PR DESCRIPTION
**Pull request by krishvoor**
This PR adds a new workflow openshift-qe-installer-aws-ovn-ipsec. Allowing us to schedule cp testing on large scale AWS Clusters

### CODE REVIEW

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.18-nightly-x86-control-plane-ipsec-120nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml [hunk 1]
```diff
@@ -201,6 +201,44 @@ tests:
...
+      SIZE_VARIANT: large
+      USER_TAGS: |
+        TicketId 382
+      ZONES_COUNT: "3"
+    test:
+    - chain: openshift-qe-control-plane
+    workflow: openshift-qe-installer-aws-ovn-ipsec
+  timeout: 8h0m0s
+- as: control-plane-ipsec-252nodes
+  cron: 0 3 22 * *
```
Comment (qiliRedHat) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml:224]: The cron is duplicated to the 120 nodes.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml [hunk 2]
```diff
@@ -201,6 +201,44 @@ tests:
     test:
     - chain: openshift-qe-node-density-cni
     workflow: openshift-qe-installer-aws
+- as: control-plane-ipsec-120nodes
+  cron: 0 3 22 * *
```
Comment (qiliRedHat) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml:205]: In this [doc](https://docs.google.com/document/d/1NbXwfZwSkMdftuKW1Ob705rQk-XHLtPtP1d1vuqF9Vo/edit#heading=h.an1n9la7teu5) Cron Rules for 'control-plane-120nodes' monthly is either 4, 18, 11, or 25. 
CC: @afcollins

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml [hunk 3]
```diff
@@ -201,6 +201,44 @@ tests:
...
+    env:
+      BASE_DOMAIN: perfscale.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "120"
+      COMPUTE_NODE_TYPE: m5.xlarge
+      EXTRA_FLAGS: --churn-duration=20m --service-latency
+      PROFILE_TYPE: reporting
+      SIZE_VARIANT: large
+      USER_TAGS: |
+        TicketId 382
+      ZONES_COUNT: "3"
```
Comment (qiliRedHat) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml:218]: Please add the following envs to specify a 16vCPU infra node size for 120 worker nodes cluster. 
```
SET_ENV_BY_PLATFORM: custom
OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: Standard_E16s_v3
```
Refer to this doc
https://docs.google.com/document/d/1sVkbU5ttS3g_GqfEeAWDzT25asvy_jKqy0qGCJ4Lfzo/edit#heading=h.x01a3v2u2rct

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml (hunk 1 above)
Comment (afcollins) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml:224]: use: `0 14 11,25 * *`

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml (hunk 2 above)
Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml:205]: Updated, will be pushing new PR

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml (hunk 1 above)
Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml:224]: Updated, pushing a new commit on the same

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml (hunk 3 above)
Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml:218]: 120, 252 nodes based on the above linked document, refer to instance size of
```
OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: r5.4xlarge
```

**Pull request merged**